android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -73,13 +80,58 @@ private void downloadFeed() {
             return;
         }
 
-        APIClient.getFacebookGraphAPI()
-                .getPosts(SharedPreferencesUtil.getString(ConstantStrings.FACEBOOK_PAGE_ID, null),
-                        getContext().getResources().getString(R.string.fields),
-                        getContext().getResources().getString(R.string.facebook_access_token))
-                .subscribeOn(Schedulers.io())
-                .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(feed -> {
+        Observable<FacebookFeed> facebookFeedObservable = null;
+        Observable<LoklakFeed> loklakFeedObservable = null;
+
+        if(SharedPreferencesUtil.getString(ConstantStrings.TWITTER_PAGE_NAME, null) != null) {
+            loklakFeedObservable = APIClient.getLoklakAPI()
+                    .getTwitterFeed(SharedPreferencesUtil.getString(ConstantStrings.TWITTER_PAGE_NAME, null), 20, "twitter")
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread());
+        }
+
+        if(SharedPreferencesUtil.getString(ConstantStrings.FACEBOOK_PAGE_ID, null) != null) {
+            facebookFeedObservable = APIClient.getFacebookGraphAPI()
+                    .getPosts(SharedPreferencesUtil.getString(ConstantStrings.FACEBOOK_PAGE_ID, null),
+                            getContext().getResources().getString(R.string.fields),
+                            getContext().getResources().getString(R.string.facebook_access_token))
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread());
+        }
+
+        if(facebookFeedObservable != null && loklakFeedObservable != null) {
```

Comment:
Space after if. Change it at other places also.
You can try `Ctrl+Alt+Shift+L`  after selecting code you have added or changed..